SDAPackage/SDA/obstacle_map.py

```diff
@@ -158,20 +159,24 @@ def generate_possible_paths(self, obstacle):
                     [obstacle.get_point()[0] + obstacle.get_radius(), obstacle.get_point()[1], obstacle.get_height() + (Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS)],
                     [obstacle.get_point()[0] - obstacle.get_radius(), obstacle.get_point()[1], obstacle.get_height() + (Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS)]
                 ]
+                #for new_point in new_attempt_pos_points:
+                #    print("New Attempt Pos Point", str(new_point))
 
                 new_paths = []
                 for new_pos_point in new_attempt_pos_points:
-                    if not self.does_path_intersect_obstacle_3d(obstacle, self.drone.get_point(), new_pos_point) and self.flight_boundary.is_point_in_bounds(new_pos_point):
+                    print(self.flight_boundary.is_point_in_bounds(new_pos_point))
+                    if not self.does_path_intersect_obstacle_2d(obstacle, self.drone.get_point(), new_pos_point) and self.flight_boundary.is_point_in_bounds(new_pos_point):
+                    #if self.flight_boundary.is_point_in_bounds(new_pos_point):
                         for recursive_new_pos_point in new_attempt_pos_points:
                             if self.flight_boundary.is_point_in_bounds(recursive_new_pos_point) and abs(recursive_new_pos_point[2] - new_pos_point[2]) < 5:
                                 if recursive_new_pos_point[0] != new_pos_point[0] or recursive_new_pos_point[1] != new_pos_point[1]:
-                                    if not self.does_path_intersect_obstacle_3d(obstacle, new_pos_point, recursive_new_pos_point) and not self.does_path_intersect_obstacle_3d(obstacle, recursive_new_pos_point, self.drone.get_waypoint_holder().get_current_waypoint()):
+                                    if not self.does_path_intersect_obstacle_2d(obstacle, new_pos_point, recursive_new_pos_point) and not self.does_path_intersect_obstacle_2d(obstacle, recursive_new_pos_point, self.drone.get_waypoint_holder().get_current_waypoint()):
                                         new_paths.append([new_pos_point, recursive_new_pos_point])
 
                 # Uncomment for DEBUGGING ONLY
                 for path in new_paths:
+                    print(self.get_path_distance(path))
                     print("Point:", str(path))
```

Comment:
I like the additional print statement. Since these comments are only for debugging, please comment them all out.

Reply:
done